api/src/dtos/listings/listings-filter-params.dto.ts

```diff
@@ -123,4 +124,13 @@ export class ListingFilterParams extends BaseFilter {
   })
   @IsArray({ groups: [ValidationsGroupsEnum.default] })
   [ListingFilterKeys.counties]?: string[];
+
+  @Expose()
+  @ApiPropertyOptional({
+    type: Array,
+    example: ['abcdef'],
+  })
+  @IsUUID(4, { groups: [ValidationsGroupsEnum.default], each: true })
+  @IsArray({ groups: [ValidationsGroupsEnum.default] })
+  [ListingFilterKeys.ids]?: string[];
```

Comment:
The new param, a list of listing IDs, that we pass to filter by the visible listings